public/style/components/_btn.scss

```diff
@@ -0,0 +1,48 @@
+.btn {
```

Comment:
This file should be renamed `_buttons.scss`. `_btn` is unnecessary shorthand here and will help people who might not otherwise be familiar with it recognize this file.